core/os/dir_access.h

```diff
@@ -34,6 +34,9 @@
 #include "core/typedefs.h"
 #include "core/ustring.h"
 
+static const String imported_files_path = "res://.godot/imported";
+static const String imported_files_path_nores = ".godot/imported";
```

Comment:
I don't think that it's correct to put this in `DirAccess`, it is not related to the low-level implementation of the filesystem API.

Reply:
I'm thinking that putting this as a public constant of `ProjectSettings` might be better (and with full caps naming as it's a constant). Might need more opinions as it's not the perfect location either (but it's more related to the project setup and properties).